commit aea8f19ee42871bcbea7fc4e8301da8f1c3c6a8d (refs/changes/35/935/2)
Author: June Yi <june.yi@samsung.com>
Date:   2016-11-24T15:27:36+00:00 (2 years, 10 months ago)
    
    joyent/sdcadm#28 add --force-data-path option for `sdcadm create portolan`

diff --git a/lib/cli/do_create.js b/lib/cli/do_create.js
index 8fa8909..479ab42 100644
--- a/lib/cli/do_create.js
+++ b/lib/cli/do_create.js
@@ -89,7 +89,8 @@ function do_create(subcmd, opts, args, cb) {
                 changes: changes,
                 progress: self.progress,
                 uuid: self.uuid,
-                skipHACheck: opts.skip_ha_check
+                skipHACheck: opts.skip_ha_check,
+                forceDataPath: opts.force_data_path
             }, function (err, plan_) {
                 plan = plan_;
                 next(err);
@@ -193,6 +194,11 @@ do_create.options = [
         type: 'bool',
         help: 'Allow create the instance even if the service is not ' +
               'HA ready.'
+    },
+    {
+        names: ['force-data-path'],
+        type: 'bool',
+        help: 'Create components in the customer data path (portolan)'
     }
 ];
 
